src/Entity/OgMembership.php

```diff
@@ -57,117 +56,37 @@
 class OgMembership extends ContentEntityBase implements ContentEntityInterface {
 
   /**
-   * @var Integer
-   *
-   * The identifier of the row.
-   */
-  protected $id;
-
-  /**
-   * @var string
-   *
-   * The UUID.
-   */
-  protected $uuid;
-
-  /**
-   * @var OgMembershipType
-   *
-   * The membership type of the current membership instance.
-   */
-  protected $type;
-
-  /**
-   * @var Integer
-   *
-   * The entity ID.
-   */
-  protected $etid;
-
-  /**
-   * @var String
-   *
-   * The entity type.
-   */
-  protected $entityType;
-
-  /**
-   * @var Integer
-   *
-   * The group ID.
-   */
-  protected $gid;
-
-  /**
-   * @var String
-   *
-   * The group type.
-   */
-  protected $groupType;
-
-  /**
-   * @var Integer
-   *
-   * The state of the membership.
-   */
-  protected $state;
-
-  /**
-   * @var Integer
-   *
-   * The unix time stamp the membership was created.
-   */
-  protected $created;
-
-  /**
-   * @var String
-   *
-   * The name of the field holding the group ID, the OG membership is associated
-   * with.
+   * {@inheritdoc}
    */
-  protected $fieldName;
+  public function getCreatedTime() {
+    return $this->get('created')->value;
+  }
 
-  /**
-   * @var Language
-   *
-   * The language of the membership.
-   */
-  protected $language;
 
   /**
-   * @param mixed $created
-   *
-   * @return OgMembership.
+   * {@inheritdoc}
    */
-  public function setCreated($created) {
-    $this->set('created', $created);
+  public function setCreatedTime($timestamp) {
+    $this->set('created', $timestamp);
     return $this;
   }
 
-  /**
-   * @return mixed
-   */
-  public function getCreated() {
-    return $this->get('created')->value;
-  }
 
   /**
    * @param mixed $entityType
    *
    * @return OgMembership.
    */
-  public function setEntityType($entityType) {
-    $this->set('entity_type', $entityType);
+  public function setMemberEntityType($entityType) {
+    $this->set('member_entity_type', $entityType);
```

Comment:
I think this one cause for the failure [over here](https://travis-ci.org/amitaibu/og/jobs/97724892#L173) since you changed the `entity_type` to `member_entity_type`